### DISCUSSION TITLE
Add a scraper for Fandom.com for Manga chapters

### DISCUSSION THREAD

**Discussion by coldiak** [maforget/ComicRackCE#67] [Plugins]
Hi
i have beside the comics library in Komga, the Manga Library, could be possible create a Manga Scraper from the website fandom.com

heres is the API info

https://community.fandom.com/api.php

here is an manga example

https://arifureta.fandom.com/wiki/Arifureta_(Main_Series)/Manga

en this is the chapter info

https://arifureta.fandom.com/wiki/Arifureta_-_Chapter_0

thanks!!

**Comment by maforget** [comment]
There are no scraper built-in the program and I will not be starting to add them and need to support them. This type of feature is better suited for a plugin. I don't discount some sort or built-in behavior for integrating scrappers when the new plugin system is implemented. But any specific implementation should be relegated to a plugin. 

I will transfer this to a discussion so others who are interested can pitch-in. I already support a lot of stuff this is too much for me.

I highly suggest that if you are interested, you try your hand at it. You can check either ComicVine Scrapper for a basis or my Amazon Scrapper for a more basic one.

**Comment by maforget** [comment]
I don't think that your idea of scrapping fandom will work. It's a collection of wikis that don't have a standard way to search/determine which pages to use and which tags to scrape in the page when every series is a different site.

We would need a single site that will be constant across series/pages. Something like mangaupdates.com seems better, but I don't think they have specific Volume/Chapter info.

**Reply by coldiak** [reply]
thanks!

**Comment by maforget** [comment]
I've created a mini scraper for mangaupdates.com. There is no gui (so check the console output `-ssc`). It only fetches the genres, but you can enable more by uncommenting some lines. 

https://github.com/maforget/ComicRack_MangaUpdateScraper

**Reply by coldiak** [reply]
i'll try it!!
thanks!!
